### DISCUSSION TITLE
IR to Bluetooth converter, will it work

### DISCUSSION THREAD

**Discussion by barbill28** [GerryDazoo/Slinger#668] [Q&A]
Verizon updated all my STB boxes and DVR with the new TV Streaming boxes and DVR box. These use Bluetooth for the remote so my current remote config in the .ini file won't work. Verizon does offer an IR box that plugs into the USB port on the Streaming TV box I'd consider buying if there's a remote config file that will work on those new boxes. Anyone using anything like that at this point or have another solution? 
Update:
The Frios TV+ boxes are Google based so thought I'd try and set them up with Google Home. I was able to add the TV+ box that connects to my Slingbox 500 to Google Home and control the TV+ box (on/off and change channels). However, when I disconnected my phone running Google Home from the local network it stopped working. Does anyone know if I added a Google Nest device and locate it in the room where my TV+ box is, if that would allow me to access and control the TV+ box remotely? I don't have any experience with Google Home devices so looking for some insight.

**Comment by skijohn877** [comment]
I'm using MOES WiFi Smart IR Remote Controllers to control my FIOS STB's. Purchased them on Amazon 

https://www.amazon.com/dp/B07QH1X7PX?ref=ppx_yo2ov_dt_b_fed_asin_title&th=1
